paymentmanager.go

```diff
@@ -0,0 +1,299 @@
+package main
```

Comment:
Unfortunately, this entire file is unnecessary from the PoV of the latest version of master.